ocp_resources/logger.py

```diff
@@ -0,0 +1,32 @@
+import logging
+import os
+
+from colorlog import ColoredFormatter
+
+
+def get_logger(name):
+    log_level = os.environ.get("OPW_LOG_LEVEL", "INFO")
```

Comment:
1. Can you please update the README?
2. What does OPW stand for?

Reply:
> What does OPW stand for?

My guess is "Openshift Python Wrapper".

Reply:
right, please suggest a better name.

Reply:
why not OCP_PYTHON_WRAPPER_LOG_LEVEL ? long but self explanatory.